ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchaserInfoHybridAdditionsTests.swift

```diff
@@ -8,95 +8,97 @@
 
 import Quick
 import Nimble
-import PurchasesCoreSwift
-import Purchases
+@testable import PurchasesHybridCommon
+@testable import RevenueCat
 
-class PurchaserInfoHybridAdditionsTests: QuickSpec {
+class CustomerInfoHybridAdditionsTests: QuickSpec {
 
     override func spec() {
         describe("rc_dictionary") {
             context("managementURL") {
                 it("contains the management url when it exists") {
-                    let purchaserInfo = PartialMockPurchaserInfo()
+                    let mockCustomerInfo = try! CustomerInfo(data: [
+                        "request_date": "2019-08-16T10:30:42Z",
+                        "subscriber": [
+                            "first_seen": "2019-07-17T00:05:54Z",
+                            "management_url": "https://revenuecat.com",
+                            "original_app_user_id": "",
+                            "subscriptions": [:],
+                            "other_purchases": [:]
+                        ]])
+
                     let urlPath = "https://revenuecat.com"
-                    let url = URL(string: urlPath)
-                    purchaserInfo.stubbedManagementURL = url
-                    
-                    let dictionary = purchaserInfo.dictionary
+
+                    let dictionary = mockCustomerInfo.dictionary
                     expect(dictionary["managementURL"] as? String) == urlPath
                 }
                 it ("contains null when the management url doesn't exist") {
-                    let purchaserInfo = PartialMockPurchaserInfo()
-                    purchaserInfo.stubbedManagementURL = nil
-                    
-                    let dictionary = purchaserInfo.dictionary
+                    let mockCustomerInfo = try! CustomerInfo(data: [
```

Comment:
FYI https://github.com/RevenueCat/purchases-ios/pull/1496 gets rid of `CustomerInfo.init(data: [String: Any])`. But it can be implemented as an extension like:
```swift
convenience init(data: [String: Any]) throws {
    let data = try JSONSerialization.data(withJSONObject: data)
    let response: CustomerInfoResponse = try JSONDecoder.default.decode(jsonData: data)

    self.init(response: response)
}
```

Reply:
Thanks for the heads up! The `CustomerInfoResponse` type isn't public, though, right? So we'd have to either copy/paste and keep two versions of it around, or hopefully eventually make a better constructor for this

Reply:
Oh never mind, `CustomerInfo` will conform to `Decodable`, so we can convert this dictionary to `Data` and use `JSONDecoder` that way.

Reply:
ohhhh I forgot about that!! That's awesome